dask/array/tests/test_routines.py

```diff
@@ -658,6 +658,49 @@ def test_unique_rand(seed, low, high, shape, chunks):
         assert_eq(e_r_d, e_r_a)
 
 
+@pytest.mark.parametrize("seed", [23, 796])
+@pytest.mark.parametrize("low, high", [
+    [0, 10]
+])
+@pytest.mark.parametrize("elements_shape, elements_chunks", [
+    [(10,), (5,)],
+    [(10,), (3,)],
+    [(4, 5), (3, 2)],
+    [(20, 20), (4, 5)],
+])
+@pytest.mark.parametrize("test_shape, test_chunks", [
+    [(10,), (5,)],
+    [(10,), (3,)],
+    [(4, 5), (3, 2)],
+    [(20, 20), (4, 5)],
+])
+@pytest.mark.parametrize("invert", [True, False])
+def test_isin_rand(seed, low, high, elements_shape, elements_chunks,
```

Comment:
I need to skip these tests for old versions of NumPy.

Reply:
Seems reasonable (especially given `isin` was introduced in NumPy 1.13).

There should be some `pytest.skip` or `@pytest.mark.skipif` lines floating around that can be used.